manifests/reposync-crd.yaml

```diff
@@ -1296,6 +1296,26 @@ spec:
                     format: int64
                     minimum: 0
                     type: integer
+                  logLevel:
+                    description: logLevel allows one to override the log verbosity
+                      of a container. ContainerLogLevelOverride is a struct with field
+                      container name and integer intended log level
+                    items:
+                      description: ContainerLogLevelOverride allows to override the
+                        log level for a container
+                      properties:
+                        containerName:
+                          description: containerName specifies the name of a container
+                            whose log level will be overridden. Must be "reconciler",
```

Comment:
`Must be one of the following:`

I don't know if they all need to be quoted or not. Check for precedent in other field descriptions.

Reply:
updated. Looks like the other field in https://github.com/GoogleContainerTools/kpt-config-sync/blob/8973d806e4bfdd3e15a5ef4817656c5e0ae2a558/pkg/api/configsync/v1beta1/resource_override.go#L72 use double quotes for all container name